dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataset/tab/DatasetFilesTab.java

```diff
@@ -232,6 +234,10 @@ public void init(DatasetVersion workingVersion) {
         guestbookResponseDialog.initForDatasetVersion(workingVersion);
     }
 
+    public boolean isFilesTabDisplayable() {
+        return !embargoAccess.isRestrictedByEmbargo(dataset) && (!workingVersion.getFileMetadatas().isEmpty() || !fileMetadatasSearch.isEmpty() || !dataset.hasEverBeenPublished());
```

Comment:
I don't think that condition after `&&` have any effect. Probably it would be the same if it would not be present

Reply:
I didn't want to interfere with present logic, but You're probably right.